src/content/docs/en/tutorials/add-content-collections.mdx

```diff
@@ -382,4 +383,4 @@ The steps below show you how to extend the final product of the Build a Blog tut
     }
     ```
 
-For the full example of the blog tutorial using content collections, see the [Content Collections branch](https://github.com/withastro/blog-tutorial-demo/tree/content-collections) of the tutorial repo.
\ No newline at end of file
+For the full example of the blog tutorial using content collections, see the [Content Collections branch](https://github.com/withastro/blog-tutorial-demo/tree/content-collections) of the tutorial repo.
```

Comment:
This doesn't makes a visual difference in Deployed site but if its a Astro Docs rule for keeping 2-3 newlines after ending a code block, then I have learned something new

Reply:
Ah, it's not a convention no (not when it doesn't affect the display) but I noticed the branch I'm currently working on  does have 2 lines, so I wonder whether this was more a copy/paste/revert thing.

We can make it 1 single blank line!